New version: CamiDiff v0.0.12

JuliaRegistrator opened the pull request:
- Registering package: CamiDiff
- Repository: https://github.com/walra356/CamiDiff.jl
- Created by: @walra356
- Version: v0.0.12
- Commit: 4a3a9bd3e764665ad52fad7b1e3f1e40a3534a04
- Reviewed by: @walra356
- Reference: https://github.com/walra356/CamiDiff.jl/commit/4a3a9bd3e764665ad52fad7b1e3f1e40a3534a04#commitcomment-151389562
- Description: A package for finite difference methods
- Release notes:
<!-- BEGIN RELEASE NOTES -->
`````
## Breaking changes

castGrid() polynom type changed to common type T
`````
<!-- END RELEASE NOTES -->

<!-- bf0c69308befbd3ccf2cc956ac8a46712550b79fc9bfb5e4edf8f833f05f4c18b06eddad8845b45beb9f45c2b8020dd62994366b6f0149e4141d71457e266cf13c8e343886809bd1b0ea70e1f4970f7da097e33b579c768c48df1027e1d67cdddb5675fe64050e2ac71267e9c3280ce70e7bf92f981f61ba56c1578084afeca4fcf7878f17bc8f8b8d4e978ef67e3d3d3665e0f624c7cb718b96e75a0baaffcccd568fc93b358e2fdc74a953e03f97e5509a4855af6227d540411a8b43c5f8ae5fd3b3806b70536366262f28ee814c7357f9d717c6a076ecffd2350e1beb02aa -->

walra356 commented:
@JuliaRegistrator register

Release notes:

## Breaking changes

- castGrid() polynom type changed to common type T

walra356 commented:
@JuliaRegistrator register

Release notes:

## Breaking changes

- castGrid() polynom type changed to common type T

walra356 commented:
[noblock]

walra356 commented:
[noblock]

@JuliaRegistrator register

Release notes:

## Breaking changes

- castGrid() polynom type changed to common type T

JuliaRegistrator committed:
New version: CamiDiff v0.0.12

UUID: 2453a446-82e3-4664-a8e3-5851c117fba8
Repo: https://github.com/walra356/CamiDiff.jl.git
Tree: 4a4783cbafa350a91ebea8325b40a46c22d490b0

Registrator tree SHA: 17aec322677d9b81cdd6b9b9236b09a3f1374c6a
